zproject/jinja2/compressors.py

```diff
@@ -0,0 +1,159 @@
+"""
+The `Jinja2Compressed` is taken from Django-Pipeline 1.2.24 and `minified_js`
+is taken from `zerver.templatetags.minified_js.py`
+"""
```

Comment:
Yes, if we upgrade to a newer version of Django-Pipeline we can use the Jinja2 extension from that version directly and `Jinja2Compressed` won't be needed.